src/pages/Login/Login.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { screen, render } from "@testing-library/react";
+import Login from "./Login";
+import { MemoryRouter } from "react-router-dom";
+import userEvent from "@testing-library/user-event"
+
+  const email = "daniela.gutperl@gmail.com"
+  const setEmail = () => {return}
+  const password = "password"
+  const setPassword = () => {return}
+  const setUserId = () => {return}
+
+it("should render elements to allow login", () => {
+  render(<MemoryRouter><Login 
```

Comment:
You can replace this `render` with `customRender` from `src/utils/testUtils.tsx`.